tests/unit-tests/mailpoet/test-class-sensei-mailpoet.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * File with class for testing Sensei MailPoet.
+ *
+ * @package sensei-tests
+ */
+
+/**
+ * Class for testing Sensei_MailPoet class.
+ *
+ * @group Sensei MailPoet
+ */
+class Sensei_MailPoet_Test extends WP_UnitTestCase {
+	use Sensei_Course_Enrolment_Manual_Test_Helpers;
+	/**
+	 * Factory object.
+	 *
+	 * @var Sensei_Factory
+	 */
+	protected $factory;
+
+	/**
+	 * Set up the test.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		$this->factory = new Sensei_Factory();
+		$mailpoet_api  = Sensei_MailPoetAPIFactory::MP();
+		new Sensei_MailPoet( $mailpoet_api );
+	}
+
+	/**
+	 * Tests manual enrolment, we are not testing anything specific to the MailPoet API.
+	 * This general test ensures the hook 'sensei_manual_enrolment_learner_enrolled' throws no error in the MailPoet class.
+	 */
+	public function testEnrolStudentNoErrors() {
```

Comment:
To keep this and the withdraw test relevant for the tested class, they could test if the student is added/removed from the related list.

Reply:
Good call, I'll add that to the feature branch PR.